bin/fba_sv1

```diff
@@ -78,24 +78,78 @@ def get_isaen(g, aen):
     )
 
 
-# AR get the updated ra,dec with proper motion
-def get_nowradec(ra, dec, pmra, pmdec, parallax, ref_epoch, new_obstime):
+# AR courtesy of DL : from astrometry.net's util/starutil_numpy.py
+def datetomjd(d):
+    d0 = datetime(1858, 11, 17, 0, 0, 0)
+    dt = d - d0
+    return dt.days + (dt.seconds + dt.microseconds / 1e6) / 86400.0
+
+
+# AR courtesy of DL : adapted from legacypipe.survey
+def mjd_to_year(mjd):
+    # mjd_to_year(57205.875) -> 2015.5
+    daysperyear = 365.25  # Julian years, by definition
+    J2000 = datetime(2000, 1, 1, 12, 0, 0, 0, tzinfo=None)
+    mjd2k = datetomjd(J2000)
+    return (mjd - mjd2k) / daysperyear + 2000.0
```

Comment:
This is likewise Time(mjd, format='mjd').jyear.

Reply:
correct; actually I "mjd" comes from "new_obstime.mjd"; so I can simply use "new_obstime.jyear", and forget about the mjd.